### PR TITLE
feat: store os in db

### PR DESCRIPTION
**Pull request by bonomat**
resolves https://github.com/get10101/meta/issues/444

### CODE REVIEW

**Comment by holzeis** [comment]
@bonomat how would that look for iOS? Does it include the version?

**Comment by bonomat** [comment]
> @bonomat how would that look for iOS? Does it include the version?

it does not include a version, only the os, i.e. one of these values: https://doc.rust-lang.org/std/env/consts/constant.OS.html

**Comment by holzeis** [comment]
> > @bonomat how would that look for iOS? Does it include the version?
> 
> it does not include a version, only the os, i.e. one of these values: [doc.rust-lang.org/std/env/consts/constant.OS.html](https://doc.rust-lang.org/std/env/consts/constant.OS.html)

With the [device_info_plus](https://pub.dev/packages/device_info_plus) (which I think we are already using) we can get the os version as well.

File: coordinator/migrations/2024-04-29-051613_add_os_to_user_table/up.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users
+    ADD COLUMN os TEXT;
```
Comment (holzeis) [inline coordinator/migrations/2024-04-29-051613_add_os_to_user_table/up.sql:2]: 🔧 Missing newline

**Review by holzeis — APPROVED** [review]

**Pull request merged**